src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -113,29 +113,35 @@ describeEachAppLayout({ sizes: ['desktop'] }, ({ theme }) => {
       expect(wrapper.findSplitPanelOpenButton()).not.toBeNull();
     });
 
-    // Not implemented on the toolbar version yet
-    (theme !== 'refresh-toolbar' ? test : test.skip)('Moves focus to slider when opened', () => {
+    test('Moves focus to slider when opened', () => {
       const { wrapper } = renderComponent(
         <AppLayout
           splitPanel={defaultSplitPanel}
           splitPanelPreferences={{ position }}
           onSplitPanelPreferencesChange={noop}
         />
       );
-      wrapper.findSplitPanelOpenButton()!.click();
+      const slidePanelTrigger =
+        theme === 'refresh-toolbar'
+          ? wrapper.find(`button[data-testid="awsui-app-layout-trigger-slide-panel"]`)
+          : wrapper.findSplitPanelOpenButton();
+      slidePanelTrigger!.click();
       expect(wrapper.findSplitPanel()!.findSlider()!.getElement()).toHaveFocus();
     });
 
-    // Not implemented on the toolbar version yet
-    (theme !== 'refresh-toolbar' ? test : test.skip)('Moves focus to open button when closed', () => {
+    test('Moves focus to open button when closed', () => {
       const { wrapper } = renderComponent(
         <AppLayout
           splitPanel={defaultSplitPanel}
           splitPanelPreferences={{ position }}
           onSplitPanelPreferencesChange={noop}
         />
       );
-      wrapper.findSplitPanelOpenButton()!.click();
+      const slidePanelTrigger =
+        theme === 'refresh-toolbar'
+          ? wrapper.find(`button[data-testid="awsui-app-layout-trigger-slide-panel"]`)
+          : wrapper.findSplitPanelOpenButton();
+      slidePanelTrigger!.click();
```

Comment:
How does this work? This click dispatches only click event without any associated events like focus, hover etc.
Let's discuss it.